source/security/incidents.html.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Incidents
+weight: 1
+last_reviewed_on: 2024-07-10
+review_in: 6 months
+---
+# Incidents
+
+An incident is an unexpected event or error which disrupts the design system as a service. These can negatively impact our users' ability to use the design system, end users' ability to use government services and the security and trust of service teams and end users. When incidents happen, we need to be able to identify them and respond to them.
+
+## Am I having an incident?
+
+It's important to be able to identify when an incident is happening so that we can start responding to it. Not noticing or responding to incidents promptly can increase their severity. At the same time, not every issue is an incident. Some useful things to look for when identifying something as an incident are:
+
+- risks or breaches to the security of users or the team
+- users not being able to use the design system, either in part or at all
+- end users not being able to use services as a result of their use of the design system or govuk-frontend
+- risks to the reputation of services or the design system
+
+Additionally, consider if a given issue needs to be solved immediately or if the design system is still able to operate securely with a given issue in place.
+
+### Examples of incidents
+
+- All or part of the design system website being unavailable
+- The design system website being defaced
+- One or more of our platform accounts being compromised eg: npm, mailchimp, bitwarden
+- Malicious code being shipped in govuk-frontend
+- Linking to a malicious resource in our documentation eg: a port of govuk-frontend that contains malicious code
+- A serious accessibility bug is shipped/discovered in govuk-frontend
+
+### Examples of things that are not incidents
+
+Note that it's still valuable to fix issues like these ones promptly but they don't need to be treated as incidents as they don't seriously impact the design system service.
+
+- Inaccurate information on the design system website
+- A non-blocking bug in govuk-frontend code
+- An accessibility bug in govuk-frontend that doesn't block users from using services
+- A user using the design system inappropriately
+- A user needing technical support with the design system or govuk-frontend
+
+## Responding to an incident
+
+Once something has been identified as an incident, the most valuable thing to do is attempt to resolve it. However, it's important to follow procedure so that we can properly document and manage incidents during and after they happen.
+
+Incidents take priority over all other work.
+
+### Let the team know
+
+The first thing to do is let the team know an incident is happening. Post a message with a summary of the incident in progress. Use '@here' so that the whole team is aware.
```

Comment:
```suggestion
The first thing to do is let the team know an incident is happening. Post a message on Slack with a summary of the incident in progress. Use '@here' so that the whole team is aware.
```